django_redis/util.py

```diff
@@ -8,7 +8,7 @@
 
 
 @python_2_unicode_compatible
-class CacheKey(str):
+class CacheKey(object):
```

Comment:
This fix seems wrong. A cache key should be a string. You should be able to call `len()`, `format`, and other stringy things on it. Doesn't this change break that?